astropy/coordinates/builtin_frames/utils.py

```diff
@@ -33,11 +33,10 @@
 
 _IERS_HINT = """
 If you need enough precision such that this matters (~<10 arcsec), you can
-download the latest IERS predictions by running:
+use the latest IERS predictions by running:
 
-    >>> from astropy.utils.data import download_file
     >>> from astropy.utils import iers
-    >>> iers.IERS.iers_table = iers.IERS_A.open(download_file(iers.IERS_A_URL, cache=True))
+    >>> iers.IERS.iers_table = iers.IERS_A.open(iers.IERS_A_URL)
```

Comment:
D'oh! @mhvk raises a good point here.  I checked, and it appears `download_file` will use the cache unconditionally if `cache=True`, even if the file has changed on the server.  This is actually bad behavior on the part of `download_file`, and I will open a separate issue for that.

In the meantime, this PR should be changed--the `IERS.open` should still accept the `cache` argument, but its default should be set to *False*.  Sorry for the confusion @sYnfo